htdocs/variants/class/ProductCombination.class.php

```diff
@@ -302,12 +302,10 @@ public function fetchByFkProductChild($productid, $donotloadpricelevel = 0)
 	 *
 	 * @param	int							$fk_product_parent	Rowid of parent product
 	 * @param	bool						$sort_by_ref		Sort result by product child reference
-	 * @return	int|ProductCombination[]						Return integer <0 KO
+	 * @return	ProductCombination[]						Return empty array if ko
```

Comment:
How do we make a difference between technical error and no error but nothing found ?